bigbone/src/main/kotlin/social/bigbone/api/entity/Filter.kt

```diff
@@ -74,7 +74,13 @@ data class Filter(
          * This is the default (and unknown values should be treated as equivalent to warn).
          */
         @SerialName("warn")
-        WARN
+        WARN,
+
+        /**
+         * Hide/blur media attachments with a warning identifying the matching filter by title.
+         */
```

Comment:
```suggestion
         * Hide/blur media attachments with a warning identifying the matching filter by title.
         * @since Mastodon 4.4.0
         */
```

Reply:
Do you know how Mastodon servers with older versions would behave if you were to e.g. create a filter with the `filterAction` set to `blur`?

Reply:
Good question. I assume that their requirement of clients to interpret unknown values sent by a server as `warn` also works the other way around - but will try to look that up.

Reply:
Oh my... turns out that not only is an unknown value for `filter_action` **not** validated or replaced with a default value by Mastodon servers < 4.4.0, but on top of that the attempt leads to a HTML error page instead of some error JSON being returned, completely breaking our parsing of that error response in `MastodonRequest.execute()`.

Thanks for catching that!

I think we should consider this to be blocked for the time being, until we implement some form of API version check, for example as suggested in #483. mastodon/mastodon#34256 increased their API version from 4 to 5 for this change.